apps/ledger-live-desktop/src/renderer/families/cosmos/TransactionConfirmFields.tsx

```diff
@@ -73,7 +73,8 @@ export const CosmosDelegateValidatorsField = ({
   const { validators } = transaction;
   const currencyId = mainAccount.currency.id;
   const { validators: cosmosValidators } = useCosmosFamilyPreloadData(currencyId);
-  const mappedValidators = mapDelegationInfo(validators || [], cosmosValidators, unit);
+  // NOTE: investigate why validators[0].amount is zero
```

Comment:
Remove comment ? (I think you fixed the issue ?)

Reply:
by passing transaction as the third argument mapDelegationInfo is able to get the amount, but it checked validators[0].amount and got 0 so that might be a bug. Although the whole flow works fine when I test it

Reply:
there is difference between what you want to delegate, and what is already delegated to the validator, maybe it's the latter that is 0 ?

Reply:
Indeed it is, not a bug then, removing the NOTE